docs/aws_setup.md

```diff
@@ -0,0 +1,53 @@
+# Deployment Documentation with GitHub Actions
+
+## Introduction
+
+This guide explains how to set up a GitHub Actions workflow to deploy an application to a remote server (e.g., AWS EC2). It also details the required environment variables for automation to work correctly.
+
+> ⚠️ **Warning:** Deployment is only triggered when a branch is merged into main, not when a commit is pushed directly to the branch.
```

Comment:
you can use one of these instead:

```suggestion
> [!IMPORTANT]  
> Deployment is only triggered when a branch is merged into main, not when a commit is pushed directly to the branch.
```

```suggestion
> [!CAUTION]  
> Deployment is only triggered when a branch is merged into main, not when a commit is pushed directly to the branch.
```

```suggestion
> [!WARNING]  
> Deployment is only triggered when a branch is merged into main, not when a commit is pushed directly to the branch.
```

Reply:
![image](https://github.com/user-attachments/assets/eeb823d9-0570-4a91-8892-9b741a9c6ce9)
Source: https://github.com/orgs/community/discussions/16925

Reply:
I think the better could be TIP, NOTE, or IMPORTANT, since Caution and Warning have a negative meaning